engine.py

```diff
@@ -248,24 +238,33 @@ def _post_app_init(self):
         Called from the main thread when all apps have initialized
         """
         # set up menu handler
-        self._menu_generator = self.tk_3dsmax.MenuGenerator(self)
-        self._add_shotgun_menu()
+        if self._max_version_to_year(self._get_max_version()) >= 2025:
+            self._menu_generator = self.tk_3dsmax.MenuGenerator2025(self)
```

Comment:
We might want to to use the name "MenuGenerator" for 2025+ and rename the previous one to "MenuGenerator_legacy" or MenuGenerator_menuMan"?

Reply:
Good call